src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.cs

```diff
@@ -40,7 +40,7 @@ internal sealed partial class CalendarData
         internal string[] saLeapYearMonthNames = null!; // Multiple strings for the month names in a leap year.
 
         // Integers at end to make marshaller happier
-        internal int iTwoDigitYearMax = 2029; // Max 2 digit year (for Y2K bug data entry)
+        internal int iTwoDigitYearMax = 2049; // Max 2 digit year (for Y2K bug data entry)
```

Comment:
@elinor-fung do you know what "// Integers at end to make marshaller happier" means? just curious

Reply:
I don't know what that is referring to - marshalling should not require integers at the end... It also doesn't look like `CalendarData` is marshalled in any way now (I assume it was once upon a time).

In any case, I think the comment should be removed.